### DISCUSSION TITLE
Risultato di una query SQLite

### DISCUSSION THREAD

**Discussion by toniobr1976** [indomus/forum#739] [HUB: Home Assistant]
Salve,
la mia home ha due schelly, uno che misura la corrente dal contatore alle utenze ed uno dai pannelli al contatore.
Non posso mettere lo shelly dalla rete al contatore perchè fisicamente impossibile.
Per conoscere i consumi reali del contatore ho inserito un'entità sensor che è la differenza tra prelevata e immessa.
Ho poi testato una query sqlite in SqliteWeb che mi restituisce la media dei prelievi MA ora non so come prelevare questo valore e mostrarlo nella plancia.
Qualche indicazione sugerimento?

**Comment by snakuzzo** [comment]
https://www.home-assistant.io/integrations/sql/
